src/color.rs

```diff
@@ -3,9 +3,25 @@
 macro_rules! colorize_bool {
     ($tf:expr) => {
         if $tf {
-            format!("{}", $tf).bright_green().to_string()
+            format!("true ").bright_green().to_string()
```

Comment:
Would prefer to use formatting parameters for alignment instead static string. eg. `"{:5}"`  or `"{:<5}"`